### PR TITLE
jalview 2.11.3.3

### PR DESCRIPTION
**Pull request by benzoid**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/j/jalview.rb [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   url "https://www.jalview.org/downloads/installers/release/Jalview-#{version.dots_to_underscores}-macos-#{arch}-java_8.dmg"
   name "Jalview"
   desc "Multiple sequence alignment editor, visualisor, analysis and figure generator"
   homepage "https://www.jalview.org/"
 
   livecheck do
-    url "https://www.jalview.org/getdown/release/install4j/1.8/updates.xml"
-    regex(/newVersion="(\d+(?:\.\d+)*)"/i)
+    url "https://www.jalview.org/getdown/release/1.8/release/build_properties"
+    regex(/^VERSION="(\d+(?:\.\d+)*)"/i)
```
Comment (krehel) [inline Casks/j/jalview.rb:15]: `build_properties` does not appear to have quotes
```suggestion
    regex(/^VERSION=(\d+(?:\.\d+)*)/i)
```

File: Casks/j/jalview.rb [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
   homepage "https://www.jalview.org/"
 
   livecheck do
-    url "https://www.jalview.org/getdown/release/install4j/1.8/updates.xml"
-    regex(/newVersion="(\d+(?:\.\d+)*)"/i)
+    url "https://www.jalview.org/downloads/installers/release/Jalview-latest-macos-x64-java_8.dmg"
+    strategy :header_match do |headers|
+      v = headers["location"][%r{/Jalview-(\d(?:_\d+)*)-macos-x64-java_8\.dmg$}i, 1]
+
+      "#{v.underscores_to_dots}"
```
Comment (p-linnane) [inline Casks/j/jalview.rb:18]:
```suggestion
    url "https://www.jalview.org/downloads/installers/release/Jalview-latest-macos-x64-java_8.dmg"
    regex(/Jalview-(\d(?:_\d+)*)-macos-x64-java_8\.dmg/i)
    strategy :header_match do |headers, regex|
      headers["location"].scan(regex).map do |match|
        match[0].tr("_", ".").to_s
      end
```

**Review by p-linnane — CHANGES_REQUESTED** [review]
Please allow maintainers to push to your fork in the future. It allows us to make quick fixes and then get things merged.

**Review by p-linnane — APPROVED** [review]

**Comment by benzoid** [comment]
> Please allow maintainers to push to your fork in the future. It allows us to make quick fixes and then get things merged.

Will do -- how is this done?

**Comment by p-linnane** [comment]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Pull request merged**